lab_1/main.py

```diff
@@ -69,7 +103,11 @@ def get_concordance(tokens: list, word: str, left_context_size: int, right_conte
     right_context_size = 3
     --> [['man', 'is', 'happy', 'the', 'dog', 'is'], ['dog', 'is', 'happy', 'but', 'the', 'cat']]
     """
-    pass
+    concordance = []
```

Comment:
consider adding checks for tests, otherwise they will fail

Reply:
I need more ifs here, why just so few? (do not rewrite, I ll ask you on October 9th if your PR ll pass)